src/core/primitives.rs

```diff
@@ -103,37 +103,44 @@ fn verify(msk: &MasterSecretKey, usk: &UserSecretKey) -> Result<(), Error> {
 
 fn G_hash(seed: &Secret<SHARED_SECRET_LENGTH>) -> Result<Scalar, Error> {
     let mut bytes = [0; 64];
```

Comment:
Is there any existing constant in `tiny-keccak` for this? (or discard this comment if this is purely internal)

Reply:
Good catch, 64 is not 256 / 8. Corrected in 18534ae.